src/awesome/console/AwesomeLinkFilter.java

```diff
@@ -162,6 +166,25 @@ public String getFileFromUrl(final String url) {
 		return results;
 	}
 
+	private List<VirtualFile> strictFilesByPaths(final List<VirtualFile> matchingFiles, final FileLinkMatch match) {
+		List<VirtualFile> strictMatchingFiles = new ArrayList<>();
+		for (VirtualFile matchedFile : matchingFiles) {
+			String generalisedFilePath = generalisePath(matchedFile.getPath());
+			String generalisedMatchPath = generalisePath(match.path);
+			if (generalisedFilePath.endsWith(generalisedMatchPath)) {
+				strictMatchingFiles.add(matchedFile);
+			}
+		}
+		return strictMatchingFiles;
+	}
+
+	private String generalisePath(final String path) {
```

Comment:
Please use american English 'generalize'

Reply:
Thanks, gonna fix it.